modules/s2sTesting.js

```diff
@@ -0,0 +1,127 @@
+import { config } from 'src/config';
+import { setS2STestingModule } from 'src/adaptermanager';
+
+var CONSTANTS = require('src/constants.json');
+const AST = CONSTANTS.JSON_MAPPING.ADSERVER_TARGETING;
+export const SERVER = 'server';
+export const CLIENT = 'client';
+
+var testing = false; // whether testing is turned on
```

Comment:
I think we agreed to avoid global mutable state if possible. This could be refactored into an instance.

Reply:
I tried to refactor to use `(getConfig('s2sConfig') || {}).testing`; each time, rather than storing this, but getConfig('s2sConfig') always returns undefined (will report the issue separately).  So in the interest of getting this out in the next release (without spending more time right now), do you mind if we leave this for now?